src/doc/src/reference/cargo-targets.md

```diff
@@ -245,10 +244,6 @@ libraries and binaries.
 > **Note**: The binary will be skipped if its name is the same as the lib
 > target.
 
-### The `plugin` field
-
-This field is used for `rustc` plugins, which are being deprecated.
```

Comment:
Sorry I wasn't clear enough. This one can be kept. Just like what we've done for `target.<triple>.ar`.

https://doc.rust-lang.org/nightly/cargo/reference/config.html#targettriplear

Reply:
Thanks for the help, it was me who didn't understand it correctly.